src/node_lkl.cc

```diff
@@ -130,3 +130,58 @@ NAN_METHOD(parseDirent64) {
 	}
 	info.GetReturnValue().Set(result);
 }
+
+NAN_METHOD(sizeOfStructLklStat) {
+	auto size = Nan::New<v8::Uint32>(
+		static_cast<uint32_t>(sizeof(struct lkl_stat))
+	);
+	info.GetReturnValue().Set(size);
+}
+
+v8::Local<v8::Value> castUint32(long unsigned int x) {
+	return Nan::New<v8::Uint32>(static_cast<uint32_t>(x));
+}
+
+v8::Local<v8::Value> castInt32(long int x) {
+	return Nan::New<v8::Int32>(static_cast<int32_t>(x));
+}
+
+v8::Local<v8::Value> timespecToMilliseconds(time_t seconds, long nanoseconds) {
+    return Nan::New<v8::Number>(
+        (static_cast<double>(seconds) * 1000) +
+        (static_cast<double>(nanoseconds / 1000000))
+    );
+}
+
+
+NAN_METHOD(parseLklStat) {
+	// Args:
+	// * Buffer buffer
+	// * function Stats
+	// Parses buffer (which should be a struct lkl_stat*) and returns a Stats
+	// object.
+	auto *s = (struct lkl_stat*) node::Buffer::Data(info[0]);
```

Comment:
why not a `reinterpret_cast<struct lkl_stat*>(node::Buffer::Data(info[0]))`? Or you can even omit the struct and do `reinterpret_cast<lkl_stat*>(node::Buffer::Data(info[0]))`